compiler/rustc_target/src/spec/targets/loongarch64_unknown_linux_ohos.rs

```diff
@@ -0,0 +1,24 @@
+use crate::spec::{Target, TargetOptions, base};
+
+pub(crate) fn target() -> Target {
+    Target {
+        // LLVM 15 doesn't support OpenHarmony yet, use a linux target instead.
```

Comment:
Question: why does here currently still support for LLVM 15 when the minimum supported version is LLVM 18?

Reply:
This line is copied from the `aarch64-unknown-linux-ohos` target. The `ohos` triple has been supported since LLVM 17, so it's time to update it. https://github.com/rust-lang/rust/pull/130809